grpc/interceptors_test.go

```diff
@@ -5,21 +5,23 @@ import (
 	"errors"
 	"log"
 	"net"
+	"net/http"
 	"strconv"
 	"strings"
 	"sync"
 	"testing"
 	"time"
 
 	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/grpc/test_proto"
+	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/probs"
+	"github.com/letsencrypt/boulder/test"
```

Comment:
Hmmm I think this may have been the result of a format-on-save in VSCode. I currently have `go-imports` configured for this. I'll try other settings to remedy this.